luaui/Widgets/cmd_extractor_snap.lua

```diff
@@ -284,6 +284,7 @@ function widget:MousePress(x, y, button)
 
 	if button == 1 and buildCmd and buildCmd[1] then
 		local alt, ctrl, meta, shift = Spring.GetModKeyState()
+		shift = (Spring.GetConfigInt("InvertQueueKey", 0) == 1) and (not shift) or shift
```

Comment:
This should be using `Spring.GetInvertQueueKey()` instead?

Reply:
Yes, that's a lot better. Idk how I missed that func, ty for pointing it out

Reply:
Fixed